README.md

```diff
@@ -218,6 +218,18 @@ await ofetch('/movies', {
 })
 ```
 
+## 💡 Adding HTTP(S) Agent
+
+If you need use HTTP(S) Agent, can add `agent` option with `https-proxy-agent`:
```

Comment:
We need to add a note that this is for Node.js only.

Reply:
Follow up: #202

Reply:
> We need to add a note that this is for Node.js only.

By this you mean server side only right? This should work in nuxt `/server` right?